plugin/src/main/java/com/yelp/codegen/KotlinGenerator.kt

```diff
@@ -142,10 +142,27 @@ class KotlinGenerator : SharedCodegen() {
     public override fun listTypeWrapper(listType: String, innerType: String) =
             "$listType<$innerType>"
 
+    public override fun listTypeUnwrapper(baseType: String) = baseType
+            .replace(" ", "")
+            .removeSuffix(">")
+            .removePrefix("List<")
```

Comment:
I'd rather prefer using [`kotlinTypeMapping["list"]`](https://github.com/Yelp/swagger-gradle-codegen/blob/master/plugin/src/main/java/com/yelp/codegen/utils/KotlinLangUtils.kt#L121) so it might be moved into `SharedCodegen` as it will be _kotlin_ indpendent

Reply:
++